PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -1460,7 +1464,13 @@ struct HfTrackIndexSkimCreator {
   using FilteredHfTrackAssocSel = soa::Filtered<soa::Join<aod::HfTrackAssoc, aod::HfSelTrack>>;
   Preslice<FilteredHfTrackAssocSel> trackIndicesPerCollision = aod::hf_track_association::collisionId;
 
-  void process( // soa::Join<aod::Collisions, aod::CentV0Ms>::iterator const& collision, //FIXME add centrality when option for variations to the process function appears
+  void processNoSkimCreator(SelectedCollisions const&)
```

Comment:
Please rename the corresponding dummy function accordingly as well.

Reply:
ok, done